nodepool/files/nodepool.yml

```diff
@@ -44,27 +32,39 @@ providers:
     hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
     image-name-format: nodepool-{image_name}-{timestamp}
     diskimages: &provider_diskimage_block
-      - name: ubuntu-xenial
+      - name: ubuntu-xenial-g1-8
+        config-drive: true
+      - name: ubuntu-trusty-g1-8
         config-drive: true
-      - name: ubuntu-trusty
+      - name: ubuntu-xenial-p2-15
+        config-drive: true
+      - name: ubuntu-trusty-p2-15
         config-drive: true
     pools: &provider_pools_block
       - name: main
         max-servers: 10
```

Comment:
This is two low for 4 labels each with min ready 3.

Reply:
It might be, but might not be. This is 10 max-servers per pool, with 3 pools - giving a total of 30 nodepool nodes at the ready at any time. I can push it up if you think it's too low, though. Any suggestions for a number? Perhaps 20 per pool?

Reply:
I always forget the pool is spread across regions... so the max available is actually max*regions.